tests/functional/vault/test_shares.py

```diff
@@ -269,10 +269,10 @@ def test_transferFrom(accounts, token, vault):
     assert vault.balanceOf(b) == token.balanceOf(vault)
 
 
-def test_do_not_issue_zero_shares(gov, token, vault, increase_pps):
+def test_do_not_issue_zero_shares(gov, strategy, token, vault, increase_pps):
     token.approve(vault, 500, {"from": gov})
     vault.deposit(500, {"from": gov})
-    strategy = increase_pps(vault, token, 500, gov)  # inflate price
+    strategy = increase_pps(vault, strategy, token, 500, gov)  # inflate price
```

Comment:
Why are you changing that?

Reply:
I updated `increase_pps` to have strategy in the argument list. Reason is that `test_progressive_withdrawal()` and `test_withdrawal_with_empty_queue()` use fresh vault and fresh strategies, so the `strategy` supplied by conftest is not the one we want to harvest in `increase_pps()`.